src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #{$gray-elements-color};
+  border-radius: 5px;
```

Comment:
This border-radius value appears in several places with the same value, making it a good candidate for a variable.